docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -101,7 +101,7 @@ The following are required options for `join_sessions`:
 |Option|Type|Description|
 |---|---|---|
 |`name`|String|The name of the allow policy|
-|`roles`|[]String|A list of names for Teleport roles that this policy applies to. Users with this role are eligible to join a Moderated Session under this policy.|
+|`roles`|[]String|A list of names of Teleport roles whose sessions this policy applies to. Moderated Sessions created by users with these roles can be joined under this policy.|
```

Comment:
Is this true for all sessions or only moderated ones?

Reply:
Just tested and confirmed that it's for all sessions.